### DISCUSSION TITLE
appimage has network errors.

### DISCUSSION THREAD

**Discussion by JorySeverijnse** [Kvan7/Exiled-Exchange-2#168] [General]
### Description
***Describe what you are experiencing (or other stuff)***
./Exiled-Exchange-2-0.1.3.AppImage does not want to startup on linux 
Kernel: Linux 6.12.4-arch1-1
Shell: zsh 5.9
Terminal: alacritty 0.14.0
WM: Sway (Wayland)
### Reproduction steps
***Steps to reproduce, if needed***
use my setup as described above
download Exiled-Exchange-2-0.1.3.AppImage
try to start it up with sudo permission

without sudo permission it does not open my file manager. When running xdg-open ~ my file manager opens without issues so xdg portal is working fine

### Screenshot
***Attach screenshot if applicable***

![image](https://github.com/user-attachments/assets/16f0668c-c71e-4ca1-94b9-81c27331f4f0)

**Comment by JorySeverijnse** [comment]
[running_with_sudo.log](https://github.com/user-attachments/files/18234369/running_with_sudo.log)
[running_without_sudo.log](https://github.com/user-attachments/files/18234374/running_without_sudo.log)

**Comment by Kvan7** [comment]
I mean this kindly but I probably will not be able to provide you too much help with this. I'll add the help-wanted tag, but I cannot help much with linux troubleshooting. If you can provide me an exact bug, I can try and fix it, but even then I'm probably not going to be able to test it so it may be easier to make a PR if you are able to nail down the exact bug.

**Comment by JoshIce** [comment]
I get the same issues when running with sudo, but when running without it works just fine. @Kvan7 in my opinion this is a config issue not related to the app, but with @JorySeverijnse environment and this issue can be closed.

@JorySeverijnse, I would suggest investigating why you can't run the app without sudo. Maybe try an AppImage integration app like Appimaged or AppImageLauncher that creates a .desktop file for you with the correct launch parameters and sets permissions correctly. 

Here's my desktop file that was created by Appimaged:

```
[Desktop Entry]
Name=Exiled Exchange 2
Exec=/home/<username>/Applications/appimaged-806-x86_64.AppImage wrap "/home/<username>/Applications/Exiled-Exchange-2-0.1.4.AppImage" --sandbox %U
Terminal=false
Type=Application
Icon=/home/<username>/.cache/thumbnails/normal/611dc7a85426edd46913f6d8c02a1621.png
StartupWMClass=Exiled Exchange 2
X-AppImage-Version=0.1.4
Categories=Utility;
X-ExecLocation=/home/<username>/Applications/Exiled-Exchange-2-0.1.4.AppImage
TryExec=/home/<username>/Applications/appimaged-806-x86_64.AppImage
Comment=/home/<username>/Applications/Exiled-Exchange-2-0.1.4.AppImage
X-AppImage-Identifier=611dc7a85426edd46913f6d8c02a1621
```

**Comment by Kvan7** [comment]
Gotcha, I'm going to convert this to a discussion then instead, and @JorySeverijnse feel free to mark a comment as a solution if it works for you (similar to stackoverflow)
